README.md

```diff
@@ -9,40 +9,40 @@ This is the officially supported PHP library for using Adyen's APIs.
+| [Fund API](https://docs.adyen.com/api-explorer/Fund/5/overview)                                            | The Fund API provides endpoints for managing the funds in the accounts on your platform. These management operations include, for example, the transfer of funds from one account to another, the payout of funds to an account holder, and the retrieval of balances in an account.                                                    | [Fund](src/Adyen/Service/Fund.php)                                      | **v5**                   |
+| [Terminal API (Cloud communications)](https://docs.adyen.com/point-of-sale/choose-your-architecture/cloud) | Our point-of-sale integration.                                                                                                                                                                                                                                                                                                          | [Cloud-based Terminal API](src/Adyen/Service/PosPayment.php)            | Cloud-based Terminal API |      |
+| [Terminal API (Local communications)](https://docs.adyen.com/point-of-sale/choose-your-architecture/local) | Our point-of-sale integration.                                                                                                                                                                                                                                                                                                          | [Local-based Terminal API](src/Adyen/Service/PosPayment.php)            | Local-based Terminal API |      |
+| [POS Terminal Management API](https://docs.adyen.com/api-explorer/postfmapi/1/overview)                    | This API provides endpoints for managing your point-of-sale (POS) payment terminals. You can use the API to obtain information about a specific terminal, retrieve overviews of your terminals and stores, and assign terminals to a merchant account or store.                                                                         | [POSTerminalManagement](src/Adyen/Service/POSTerminalManagementApi.php) | **v1**                   |
+| [Disputes API](https://docs.adyen.com/api-explorer/Disputes/30/overview)                                   | You can use the [Disputes API](https://docs.adyen.com/risk-management/disputes-api) to automate the dispute handling process so that you can respond to disputes and chargebacks as soon as they are initiated. The Disputes API lets you retrieve defense reasons, supply and delete defense documents, and accept or defend disputes. | [Disputes](src/Adyen/Service/DisputesApi)                               | **v30**                  |
```

Comment:
minor, but otherwise the link gives a 404
```suggestion
| [Disputes API](https://docs.adyen.com/api-explorer/Disputes/30/overview)                                   | You can use the [Disputes API](https://docs.adyen.com/risk-management/disputes-api) to automate the dispute handling process so that you can respond to disputes and chargebacks as soon as they are initiated. The Disputes API lets you retrieve defense reasons, supply and delete defense documents, and accept or defend disputes. | [Disputes](src/Adyen/Service/DisputesApi.php)                               | **v30**                  |
```